src/main/java/frc/team2412/robot/util/motorcontroller/BrushlessSparkMaxController.java

```diff
@@ -15,6 +15,9 @@ public BrushlessSparkMaxController(int id, MotorControlMode mode) {
 		this.motor = new CANSparkMax(id, MotorType.kBrushless);
 		this.motorPID = motor.getPIDController();
 		this.mode = mode;
+		motorPID.setPositionPIDWrappingEnabled(true);
+		motorPID.setPositionPIDWrappingMaxInput(150 / 7);
```

Comment:
How did 150/7 come about? Looking at the [REV example](https://github.com/REVrobotics/MAXSwerve-Java-Template/blob/main/src/main/java/frc/robot/Constants.java), this should the encoder value of one full turn (2 PI, in the REV example).